clients/unshaded/src/main/java/tachyon/client/file/FileOutStream.java

```diff
@@ -97,8 +101,21 @@ public FileOutStream(long fileId, OutStreamOptions options) throws IOException {
     }
     mClosed = false;
     mCanceled = false;
-    mHostname = options.getHostname();
     mShouldCacheCurrentBlock = mTachyonStorageType.isStore();
+
+    // get default policy
+    Class<? extends FileWriteLocationPolicy> locationPolicyClass = ClientContext.getConf()
```

Comment:
shouldn't we only load default policy when `options.getLocationPolicyClass() == null`?